Gemfile

```diff
@@ -58,7 +58,7 @@ group :acceptance do
     gem 'beaker', *location_for(beaker_version)
   else
     #use the pinned version
-    gem 'beaker', '~>2.4'
+    gem 'beaker', :git => 'git://github.com/puppetlabs/beaker.git', :ref => '3e2e1e6d3e0234612c3a6af7dbfac06c797676f4'
```

Comment:
@mullr sorry to backtrack, but could we put a note here explaining the reason for the pin? Other than that this looks good.